next-tavla/src/Admin/scenarios/Edit/index.tsx

```diff
@@ -18,6 +21,13 @@ function Edit({
     documentId: string
 }) {
     const [settings, dispatch] = useReducer(settingsReducer, initialSettings)
+    const [showLink, setShowLink] = useState(false)
```

Comment:
Jeg tenker vi burde gjøre dette på en annen måte så vi ikke tvinger brukeren til å lagre settings for å se "Vis tavla" linken.

Kanskje at linken alltid er synlig i første omgang og åpnes i en annen fane, evt. undersøke om en "Du har ikke lagret endringene dine, er du sikker?"-popup gir mening?

Reply:
Hva er egentlig hensikten med å skjule linken før man lagrer? Jeg tenker det alltid er relevant å kunne se linken.
 
Kanskje vi kunne laget en separat indikator på siden som viser om du har endringer som ikke er lagret enda?

Reply:
Jeg syntes på generelt grunnlag at "Se avgangstavle" burde åpnes i en ny fane